src/stock-status-ribbon.html

```diff
@@ -11,7 +11,7 @@
     >
       <span>
         <span
-          ng-bind="$ctrl.data[bar.id].value | number: 0"
+          ng-bind="$ctrl.data[bar.id].value | number: ($ctrl.config.decimalPlaces || 0)"
```

Comment:
We might consider replacing `|| 0` with a default config that the component specifies, which is extended (with values overwritten where specified) by the config passed in by the consumer (`$ctrl.config`).